src/gt4py/ir/utils.py

```diff
@@ -70,9 +70,6 @@ def make_field_ref(name: str, offset=(0, 0, 0), *, axes_names=None):
 
 
 def make_axis_interval(bounds: tuple, *, offset_limit: int = 2):
```

Comment:
I guess offset_limit is not needed anymore and thus it can be deleted, right?

```suggestion
def make_axis_interval(bounds: tuple):
```